source/real_locations.rst

```diff
@@ -0,0 +1,116 @@
+Checking the people in a location (SIR)
+=======================================
+
+What we were doing
+------------------
+
+While calculating the number of people who could potentially infect a person, we did the following:
```

Comment:
This is a good way to start, but could you add one more line saying something along the lines of

> In our earlier SIR model, the method that we used to decide if a person would be infected or not was the following: we retrieved the type of location that the person in question was supposed to be in from their schedule, and then computed the number of people who could potentially infect them using the following function:

or something like that. Basically, mention where we got `node` and `placeType` from.